compose/cli/main.py

```diff
@@ -76,11 +87,15 @@ def main():
     except NeedsBuildError as e:
         log.error("Service '%s' needs to be built, but --no-build was passed." % e.service.name)
         sys.exit(1)
+    except PluginError as e:
+        log.error(e.msg)
+        sys.exit(1)
     except (errors.ConnectionError, StreamParseError):
         sys.exit(1)
 
 
 def dispatch():
+    plugin_manager = PluginManager(get_plugin_dir())
```

Comment:
Should this be after setup_logging() in case you need to log something from the PluginManager class setup?